app/controllers/feedback_controller.rb

```diff
@@ -0,0 +1,21 @@
+class FeedbackController < ApplicationController
+  def new
+    @feedback = Feedback.new(url: request.referrer)
+    @form = FeedbackForm.new(@feedback)
+  end
+
+  def create
+    @form = FeedbackForm.new(Feedback.new)
+    if @form.validate(feedback_params)
+      @form.save
```

Comment:
It won't because its valid. This is programming by contract and the recommended pattern for reform.